bundles/XliffBundle/src/ExportDataExtractorService/DataExtractor/DocumentDataExtractor.php

```diff
@@ -50,18 +50,29 @@ public function extract(TranslationItem $translationItem, string $sourceLanguage
         }
 
         $this
-            ->addDoumentEditables($document, $result)
+            ->addDocumentEditables($document, $result)
             ->addSettings($document, $result);
 
         return $result;
     }
 
     /**
-     *
-     *
-     * @throws \Exception
+     * @deprecated
      */
     protected function addDoumentEditables(Document $document, AttributeSet $result): DocumentDataExtractor
+    {
+        trigger_deprecation(
+            'pimcore/pimcore',
+            '11.2',
```

Comment:
oh....i had a mix up, actually we are releasing `admin-ui-classic-bundle` `1.1`, not pimcore `11.1`, so my previous comment was wrong and should be
```suggestion
            '11.1',
```